docs/releasing.md

```diff
@@ -78,8 +78,8 @@ Cutting a Release
 
     ```
     sed -i "" \
-      "s/VERSION_NAME=.*/VERSION_NAME=$RELEASE_VERSION/g" \
-      gradle.properties
+      "s/version = '.*'/version = '$RELEASE_VERSION'/g" \
```

Comment:
This suggests we should consider moving it to it's own file, or calculating it from the branch, tag name.

https://github.com/nemerosa/versioning does the latter.

Reply:
I think I wanna punt on that for a follow-up. (I’m anxious about the complexity cost / benefit of such things, especially as the versioning thing is more code that itself needs to be kept up to date.)